pandas/core/ops.py

```diff
@@ -582,6 +588,150 @@ def _get_op_name(op, special):
 DataFrame.{reverse}
 """
 
+_flex_comp_doc_FRAME = """
+Flexible wrappers to comparison operators (`eq`, `ne`, `le`, `lt`, `ge`, `gt`).
+
+Equivalent to `==`, `=!`, `<=`, `<`, `>=`, `>` with support to choose
+axis (rows or columns) and level for comparison.
+
+Parameters
+----------
+other : constant, list, tuple, ndarray, Series, or DataFrame
+    Any single or multiple element data structure, or list-like object.
+axis : int or str, optional
+    Axis to target. Can be either the axis name ('index', 'rows',
+    'columns') or number (0, 1).
+level : int or name
+    Broadcast across a level, matching Index values on the passed
+    MultiIndex level.
+
+Returns
+-------
+result : DataFrame of bool
+    Result of the comparison.
+
+See also
```

Comment:
I think the `A` should be upper case

Reply:
I see no current docs page that has `Also` as title case.